app/Main.hs

```diff
@@ -58,7 +59,7 @@ main = do
             ExitSuccess -> Nothing
             ExitFailure n -> Just n
       errorCodes <-
-        mapMaybe selectFailure <$> mapM (formatOne' . Just) (sort xs)
+        mapMaybe selectFailure <$> pooledMapConcurrentlyN optThreads (formatOne' . Just) (sort xs)
```

Comment:
As most of Ormolu should be CPU-bound, consider adding `-threaded -with-rtsopts=-N`: 

https://github.com/tweag/ormolu/blob/25b04d45b4f3e8db81bc4a863f2fededda7bc384/ormolu.cabal#L155